gridstatus/tests/test_pjm.py

```diff
@@ -13,6 +13,51 @@
 from gridstatus.tests.decorators import with_markets
 
 
+# Fixtures for sample data
+@pytest.fixture
+def sample_5min_solar_forecast_data():
+    fixture_path = "gridstatus/tests/fixtures/pjm/pjm_5min_solar_forecast_sample.json"
```

Comment:
> These fixtures are probably a bit large to be checking into source control, but that is also a bit in the eye of the beholder.

what would be the other easy option? a public s3 bucket?

that seems like it would be pretty straightforward to set up.

Reply:
yep I think that's the move. Probably in the future the test harness setup would check if you have the latest fixture zipped file locally, if so use it, if not download it from s3. (And we add that directory to the gitignore).